lib/internal/errors.js

```diff
@@ -129,6 +129,8 @@ E('ERR_INVALID_OPT_VALUE',
   (name, value) => {
     return `The value "${String(value)}" is invalid for option "${name}"`;
   });
+E('ERR_INVALID_REPL_EVAL_CONFIG',
+  'Cannot specify both breakEvalOnSigint and eval for REPL');
```

Comment:
I think we agreed to use quotation marks around parameter names in error message (`'Cannot specify both "breakEvalOnSigint" and "eval" for REPL'`), see #1220 (or literally three lines above in `errors.js`). I am not 100% sure whether this applies here, can someone confirm?

Anyway, that would make this PR semver-major, so feel free to ignore it in order to get your PR in soon :)